### PR TITLE
Move VersionHelper to VersionHandler

### PR DESCRIPTION
**Pull request by migoox**
In one of my PRs in mpzpn, @lukaszgarstecki suggested to add a support for not paginated sorted query in `Pagination.cs` and to move `VersionHelper` content to `VersionHandler`. Here is the PR I've promised to make 😊

### CODE REVIEW

File: backend/src/Examples/ExampleApp.Examples.Api/Helpers/.gitkeep
Comment (jakubfijalkowski) [inline backend/src/Examples/ExampleApp.Examples.Api/Helpers/.gitkeep:1]: Let's remove it if not needed anymore.

**Review by jakubfijalkowski — CHANGES_REQUESTED** [review]
I would request splitting this PR into two - one for version helper changes, and the other for "not paginated sorted query". These are not related to each other.

Moreover, I am reluctant to add the not-paginated queries. What is the use case?

**Comment by Saancreed** [comment]
> What is the use case?

I see one: when you know your result will never be large enough for pagination be valuable, but you want to sort by some property where the exact sorting rules are hard to reimplement on every client. For example, you might ask for a list of teams in some competition ordered by points in general ranking but you _also_ want teams with equal amount of points to be ordered by their number of wins, and if that's also equal then by something else. The client might not even get the data needed to resolve tie-breaks correctly in the result DTO.

**Comment by jakubfijalkowski** [comment]
I will be more precise - _why_ do we want the query structure to be predefined in the ExampleApp?

My main problem is that this is a somewhat rare use case. It needs to be thought out to be used because most of the time it will evolve into a problem. We don't gain that much with this kind of query being unified in the structure, because the structure itself is rather simple. Moreover, clients won't benefit from it being unified because they will handle it the same way as a normal query (because to them, this _is_ a normal query). Also, it is not used in the project at all. And the naming is complex.

**Comment by Saancreed** [comment]
Ah yeah, there is a use case but attempting to unify/generalize this may be quite pointless indeed :sweat_smile:

**Comment by migoox** [comment]
> I will be more precise - _why_ do we want the query structure to be predefined in the ExampleApp?
> 
> My main problem is that this is a somewhat rare use case. It needs to be thought out to be used because most of the time it will evolve into a problem. We don't gain that much with this kind of query being unified in the structure, because the structure itself is rather simple. Moreover, clients won't benefit from it being unified because they will handle it the same way as a normal query (because to them, this _is_ a normal query). Also, it is not used in the project at all. And the naming is complex.

My reasoning behind the not paginated sorted query is that we can use some complicated sorting logic for queries that are paginated and for those that are not (as @Saancreed has also said). You have convinced me that it is a rare use case, which unnecessarily complicates the generated code.

I will change the PR so it only includes the changes to the VersionHandler.

**Review by Saancreed — APPROVED** [review]

**Review by jakubfijalkowski — APPROVED** [review]

**Pull request merged**